### PR TITLE
Feature/20 timer formatting

### PR DESCRIPTION
**Pull request by annnuuuh**
What (if any) features are you implementing?
  - Timer Start Button functionality -- when a user completes all fields and clicks start activity, the inputted time will appear at the top of the screen and will start to count down after the user clicks the start button. Once the timer reaches zero, the button will say complete.
  
What (if anything) did you refactor?
  - refactored the html to change the Timer Start from a div to a button and then refactored css to match. 
  - Refactored startTimer method in Activity.js to countdown and end on '00:00'
  
Were there any issues that arose?
- We originally attempted to call the method in our event handler and after several unsuccessful attempts, we realized we'd need a function to appropriately handle the method.

Is there anything needed from your teammates?
  - We still need to refactor the timer display so that zeroes are present when a minute or second has just one digit. 
  
Next intended features
  - Iteration 4 features, Button Color Changing styling.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -18,7 +18,7 @@ <h1 id="left-section-header">New Activity</h1>
 
       <!-- Hide this 'formview' with classlist.add 'hidden'  on start activity button event click-->
       <span class ="" id="formView">
```
Comment (pthan1) [inline index.html]: Does this need a class?

File: Activity.js [hunk 2]
```diff
@@ -6,12 +6,10 @@ class Activity{
         this.seconds = seconds;
         this.completed = false;
         this.id = `ID${Date.now()}`;
-        // this.countdownTimeInMS = (this.minutes*60000)+(this.seconds*1000);
-        // this.duration = countdownTimeInMS/1000
         this.running = false
     }
 
-    startTimer(){
+    startTimer(event){
```
Comment (natesheridan) [inline Activity.js]: I dont think we need the event parameter here as we aren't using it anywhere else in the function.

File: index.html [hunk 3]
```diff
@@ -73,9 +73,9 @@ <h2>Seconds</h2>
           <!-- Update activityType.innerText to activity category -->
           <p id="activityType">Activity Type</p>
           <!-- input parsed countdown time to countdownTimerText on each interval -->
-          <span id="countdownTimerText">05:00</span>
+          <span id="countdownTimerText"></span>
           <!-- classlist.add on this with the below VV format based on category selected-->
-          <div id="startTimerButton" class="meditate-start-button">START</div>
+          <button id="startTimerButton" class="meditate-start-button">START</button>
```
Comment (pthan1) [inline index.html]: Change class name to something more generic

File: main.js [hunk 4]
```diff
@@ -13,10 +13,19 @@ var timeErrMsg = document.querySelector('.time-err-msg');
 var studyBtn = document.getElementById('study');
```
Comment (pthan1) [inline main.js:13]: Organize variables in groups

File: main.js [hunk 5]
```diff
@@ -26,15 +35,15 @@ function switchLeftDisplay(){
...
+  categoryChoice2 = newCard;
   leftSectionHeader.innerText = "Current Activity";
-  categoryName.innerText = activityInput.value;  
+  categoryName.innerText = activityInput.value;
   activityCards.push(newCard);
 }
+// leftSection.addEventListener('click', categoryChoice2.startTimer);
 
 function onlyNumberKey(evt) {
   // Only ASCII character in that range allowed
```
Comment (pthan1) [inline main.js]: Remove comments

File: index.html (hunk 1 above)
Comment (annnuuuh) [reply index.html]: deleted class!

File: Activity.js (hunk 2 above)
Comment (annnuuuh) [reply Activity.js]: deleted event param!

File: index.html (hunk 3 above)
Comment (annnuuuh) [reply index.html]: changed to "start-button"

**Review by pthan1 — COMMENTED** [review]
Thanks

File: main.js (hunk 4 above)
Comment (annnuuuh) [reply main.js:13]: We can probably tackle this at the end?

File: main.js [hunk 6]
```diff
@@ -26,15 +35,15 @@ function switchLeftDisplay(){
 function submitActivity(event) {
   event.preventDefault();
   switchLeftDisplay();
-
   var categoryChoice = document.querySelector('input[name="activity_categories"]:checked');
-
-  var newCard = new Activity(categoryChoice.value, activityInput.value, minutesInput.value, secondsInput.value);  
-
+  var newCard = new Activity(categoryChoice.value, activityInput.value, minutesInput.value, secondsInput.value);
+  countdownText.innerText = (`${minutesInput.value}:${secondsInput.value}`);
```
Comment (pthan1) [inline main.js:40]: Remove tab

File: main.js (hunk 5 above)
Comment (annnuuuh) [reply main.js]: removed!

File: main.js [hunk 7]
```diff
@@ -13,10 +13,19 @@ var timeErrMsg = document.querySelector('.time-err-msg');
...
 
 submitForm.addEventListener("click", validate);
+startTimerBtn.addEventListener("click", beginCountDown);
+
+function beginCountDown() {
+  categoryChoice2.startTimer();
+}
 
 function switchLeftDisplay(){
    timerView.classList.toggle("hidden");
```
Comment (pthan1) [inline main.js:31]: We use classList.add on line 59.  I think we should use one or the other in the codebase.

File: main.js [hunk 8]
```diff
@@ -26,15 +35,15 @@ function switchLeftDisplay(){
...
-
+  var newCard = new Activity(categoryChoice.value, activityInput.value, minutesInput.value, secondsInput.value);
+  countdownText.innerText = (`${minutesInput.value}:${secondsInput.value}`);
+  categoryChoice2 = newCard;
   leftSectionHeader.innerText = "Current Activity";
-  categoryName.innerText = activityInput.value;  
+  categoryName.innerText = activityInput.value;
   activityCards.push(newCard);
 }
+// leftSection.addEventListener('click', categoryChoice2.startTimer);
```
Comment (pthan1) [inline main.js]: Remove

**Pull request merged**